plugins/themes/default/styles/pages/catalog.less

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file  plugins/themes/default/styles/pages/catalog.less
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @brief Styles applying to catalog page
+ * @link templates/frontend/pages/catalog.tpl
+ */
+.page_catalog {
+	.pkp_series_nav_menu{
+
+		border-top: @bg-border;
+		margin: (@triple * 2) -@base;
+		padding: @double;
+		margin-bottom: -(@triple * 2);
+
+		> h2 {
```

Comment:
You can drop the `>` from this and the `> ul` below. The `.pkp_series_nav_menu` is specific enough and removing these will make the CSS work even if the HTML gets shifted around a bit in the future.